main/management/commands/remove_invalid_matches.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management.base import BaseCommand, CommandError
+from main.models import Match
+from main.helpers import remove_invalid_matches, get_invalid_matches
+
+
+class Command(BaseCommand):
+    help = 'Removes invalid matches'
```

Comment:
Minor: A small coding style that we are adapting in the team is that user-facing strings are in double quotes and internal strings are in single quotes. This way we can tell easily what the user will see and we don't need to argue over double and single quotes.

So this is a double, as it will show up in a help message when you say
```
./manage.py remove_invalid_matches --help
```